src/main/java/org/broadinstitute/hellbender/utils/test/SerializationTestUtils.java

```diff
@@ -0,0 +1,29 @@
+package org.broadinstitute.hellbender.utils.test;
+
+
+import org.apache.spark.SparkConf;
+import org.apache.spark.serializer.KryoSerializer;
+import org.apache.spark.serializer.SerializerInstance;
+import org.broadinstitute.hellbender.utils.Utils;
+import scala.reflect.ClassTag;
+import scala.reflect.ClassTag$;
+
+public class SerializationTestUtils {
```

Comment:
I figured it would be useful to have this available in public, but I can just move it back to protected.